### ISSUE TITLE
Show child tasks if the issue is a parent task or contains child tasks

### ISSUE THREAD

**Issue by jackyliang** [yanjost/redmine-cli#8]
Currently if you use `redmine issue [some task number]` on a parent task, it only shows the title of the parent task.

Develop a feature where if the issue contains child tasks, display all child tasks as well.

**Comment by yanjost** [comment]
Good idea ! I suggest also that the child issues are indented and/or shown as a tree

**Comment by allan-simon** [comment]
i will look into that, there's certainly some nice python library to display "termina-based tree"

**Comment by jackyliang** [comment]
Yes, precisely. This is what I had in mind: http://www.computerhope.com/unix/tree.htm

.
├── config.dat
├── data
│   ├── data1.bin
│   ├── data2.sql
│   └── data3.inf
├── images
│   ├── background.jpg
│   ├── icon.gif
│   └── logo.jpg
├── program.exe
└── readme.txt

**Comment by yanjost** [comment]
I have found this module : https://pypi.python.org/pypi/asciitree/ but I don't known if it is worth integrating it

**Comment by jackyliang** [comment]
That looks good. The things I think to consider on a user experience level is: 

1. What would you do if the parent task has like 200+ tasks? I think at that point, if all 200 tasks are printed and nested, it might as well be as useless as not being printed, since that's too much.

2. What info do you plan to print per issue? The existing format of the title? In my opinion something like `#[Task Number] [Title]` would be the best

Thanks a lot =D

**Comment by yanjost** [comment]
1. maybe set a default limit ?
2. We have to think of a default format for all outputs

**Comment by allan-simon** [comment]
issue a PR #9 ,  

I don't it makes sense to put a limit because it becomes easily tricky to define what does the limit , limit  for the tree view , and anyway you "got them all" from the api response, and adding a default limit needs to have a option to display more etc.  

at the opposite displaying everything does not cost more, because well you already got everything in the API response anyway, and in terminal i think it's really the job of a pager ( more / less / most ) to paginate  (or  tail/head )
